node_modules/primer-states/build/build.css

```diff
@@ -1 +0,0 @@
-.state{display:inline-block;padding:4px 8px;font-weight:600;line-height:20px;color:#fff;text-align:center;background-color:#999;border-radius:3px}.state-open,.state-proposed,.state-reopened{background-color:#6cc644}.state-merged{background-color:#6e5494}.state-closed{background-color:#bd2c00}.state-renamed{background-color:#fffa5d}
```

Comment:
If you need `primer-states` then include the `primer-labels` package. States were moved into the labels package, and the labels package was added to the product package. If you don't need states then you can delete all these files and references to them.

Reply:
5473cb94bc941f37e03eb4599db01ba82030508a